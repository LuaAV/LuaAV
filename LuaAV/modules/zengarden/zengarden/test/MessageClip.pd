#N canvas 221 283 450 300 10;
#X obj 38 223 clip;
#X obj 38 10 loadbang;
#X obj 106 10 bng 15 250 50 0 empty empty empty 17 7 0 10 -262144 -1
-1;
#X msg 347 122 87;
#X obj 38 268 print;
#X msg 306 122 23;
#X msg 265 122 11;
#X msg 164 122 60;
#X msg 200 122 44 55 66;
#X obj 38 46 t b b b b b b b;
#X msg 102 122 77 66 55;
#X obj 38 122 44 66 55;
#X connect 0 0 4 0;
#X connect 1 0 9 0;
#X connect 2 0 9 0;
#X connect 3 0 0 2;
#X connect 5 0 0 1;
#X connect 6 0 0 0;
#X connect 7 0 0 0;
#X connect 8 0 0 0;
#X connect 9 0 11 0;
#X connect 9 1 10 0;
#X connect 9 2 7 0;
#X connect 9 3 8 0;
#X connect 9 4 6 0;
#X connect 9 5 5 0;
#X connect 9 6 3 0;
#X connect 10 0 0 0;
#X connect 11 0 0 0;
