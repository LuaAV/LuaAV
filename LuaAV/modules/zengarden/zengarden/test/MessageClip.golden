[@ 0.000ms] print: 23
[@ 0.000ms] print: 55
[@ 0.000ms] print: 60
[@ 0.000ms] print: 55
[@ 0.000ms] print: 66
